napari/view_layers.py

```diff
@@ -263,126 +263,108 @@ def imshow(
-        represented by a list of array like image data. If not specified by
-        the user and if the data is a list of arrays that decrease in shape
+        represented by a list of array-like image data. If not specified by
+        the user and if the data is a list of arrays that decrease in shape,
         then it will be taken to be multiscale. The first image in the list
         should be the largest. Please note multiscale rendering is only
         supported in 2D. In 3D, only the lowest resolution scale is
         displayed.
-    cache : bool
-        Whether slices of out-of-core datasets should be cached upon
-        retrieval. Currently, this only applies to dask arrays.
+    name : str or list of str
+        Name of the layer.
+    opacity : float or list
+        Opacity of the layer visual, between 0.0 and 1.0.
     plane : dict or SlicingPlane
         Properties defining plane rendering in 3D. Properties are defined in
         data coordinates. Valid dictionary keys are
         {'position', 'normal', 'thickness', and 'enabled'}.
-    experimental_clipping_planes : list of dicts, list of ClippingPlane, or ClippingPlaneList
-        Each dict defines a clipping plane in 3D in data coordinates.
-        Valid dictionary keys are {'position', 'normal', and 'enabled'}.
-        Values on the negative side of the normal are discarded if the plane is enabled.
-    custom_interpolation_kernel_2d : np.ndarray
-        Convolution kernel used with the 'custom' interpolation mode in 2D rendering.
+    projection_mode : str
+        How data outside the viewed dimensions, but inside the thick Dims slice will
+        be projected onto the viewed dimensions. Must fit to cls._projectionclass
+    rendering : str or list of str
+        Rendering mode used by vispy. Must be one of our supported
+        modes. If a list then must be same length as the axis that is being
+        expanded as channels.
+    rgb : bool, optional
+        Whether the image is RGB or RGBA if rgb. If not
+        specified by user, but the last dimension of the data has length 3 or 4,
+        it will be set as `True`. If `False`, the image is interpreted as a
+        luminance image.
+    rotate : float, 3-tuple of float, n-D array or list.
+        If a float, convert into a 2D rotation matrix using that value as an
+        angle. If 3-tuple, convert into a 3D rotation matrix, using a yaw,
+        pitch, roll convention. Otherwise, assume an nD rotation. Angles are
+        assumed to be in degrees. They can be converted from radians with
+        'np.degrees' if needed.
+    scale : tuple of float or list of tuple of float
+        Scale factors for the layer.
+    shear : 1-D array or list.
+        A vector of shear values for an upper triangular n-D shear matrix.
+    translate : tuple of float or list of tuple of float
+        Translation values for the layer.
+    visible : bool or list of bool
```

Comment:
From what I understand the intention is to have the docstring in alphabetical order. Is there a reason that viewer and title are not in alphabetical order?

Reply:
yes. They are passed to Viewer constructor, not add_image, so their semantic is different.